components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -61,21 +61,244 @@ Default.args = {
 
 export const NestedMultiline = Template.bind({});
 NestedMultiline.args = {
+	items: [
+		{
+			iconName: "FolderOpen",
+		},
+		{
+			label: "Sub Item",
+		},
+		{
+			label: "Trend",
+			isDragged: true,
+		},
+		{
+			label: "January 2019 Assets",
+		},
+	],
+	variant: "multiline",
+};
+
+/**
+ * Stories for the MDX "Docs" only.
+ * Based off of the base `Template` which does not have conditional Chromatic-only markup.
+ */
```

Comment:
The JS-doc syntax in CSF files is for the automated docs and shouldn't include code-related notes; for code notes, you can use `/*` without the double `**` in order to prevent their rendering.

Reply:
Oh gosh, I didn't realize they might be rendering somewhere!! thanks for catching that.

Reply:
@castastrophe Should we name this heading "Story arguments", "Story args", "Story arg types" or something like that, instead of "Properties"? I'm wondering if "Properties" if the best naming for what these are and what Storybook calls them. This would apply to our auto docs template and custom MDX files.